.gitignore

```diff
@@ -1,8 +1,7 @@
-composer.lock
 vendor
 node_modules
 dist
-
+.DS_Store
```

Comment:
Consider adding `.DS_Store` to a global .gitignore file: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Clever, should I remove dsstore from this file?

Reply:
Yes, please do that. For reference, my `~/.gitignore` contains this:

```
*~
.DS_Store
.dir-locals.el
_misc
.sass-cache
TAGS
TestCommand.php
_*.php
*.debug

playbook.retry
.ac-php-conf.json
.local.env.local
mutagen.yml.lock
```

It's left as an exercise to the reader to pick only the stuff that makes sense for it (the reader).